tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1120,8 +1149,10 @@ public:
 
 template <typename ResultType>
 class R__CLING_PTRCHECK(off) SumHelper : public RActionImpl<SumHelper<ResultType>> {
-   const std::shared_ptr<ResultType> fResultSum;
+   std::shared_ptr<ResultType> fResultSum;
    Results<ResultType> fSums;
+   Results<ResultType> fCompensations;
+   unsigned int fNSlots;
```

Comment:
why do we need to save the fNSlots here?

Reply:
Ah, thank you, that has become obsolete and can be removed, I believe.